README.md

```diff
@@ -133,7 +226,7 @@ jobs:
         with:
           connect-host: <Your Connect instance URL>
           connect-token: ${{ secrets.OP_CONNECT_TOKEN }}
-
+          service-account-token: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
```

Comment:
Adding both configurations in this example is not a great practice. I'd rather provide two examples: one with Connect and one with Service Accounts instead.

Another solution is to just remove it from this example.

Reply:
removed